hazelcast/src/main/java/com/hazelcast/query/impl/AttributeIndexRegistry.java

```diff
@@ -235,6 +237,50 @@ public boolean canEvaluate(Class<? extends Predicate> predicateClass) {
             return delegate.evaluate(predicate);
         }
 
+        @Override
+        public Iterator<QueryableEntry> getSqlRecordIterator() {
```

Comment:
As far as I understand these getSqlRecord* methods are mostly for completeness here and not really used, right?

Reply:
I was not aware that they will be not used, because didn't understand the purpose of this class good enough. 
After reading docs and running tests, it is apparent that these implementations are not needed. Removed them.